integration/fixtures/osio_provider_config.toml

```diff
@@ -0,0 +1,17 @@
+logLevel = "ERROR"
+defaultEntryPoints = ["http"]
+[entryPoints]
+  [entryPoints.http]
+  address = ":8000"
+  [entryPoints.traefik]
+  address = ":7888"
+
+checkNewVersion = false
+
+[api]
+entryPoint = "traefik"
+
+[osio]
+refreshSeconds = 3
+clusterAPI = "http://127.0.0.1:9091/api/clusters"
+tokenAPI = "http://127.0.0.1:9091/api/token"
```

Comment:
@aslakknutsen This is configuration for 'osio' traefik backend.  Please add above variable in traefik.toml file in f8-oso-proxy project.  As decided, refreshSeconds = 900 (i.e. 15 mins) for production.